recipes/R/Retail - GradientBoosting/R/applicationScorer.R

```diff
@@ -60,6 +60,10 @@ applicationScorer <- setRefClass("applicationScorer",
       # Data Preparation/Feature Engineering
       #########################################
       timeframe <- configurationJSON$timeframe
+      if(any(names(df) == '_id')) {
+        utils <- Utils$new()
+        df = utils$mapFields(df)
+      }
```

Comment:
Exact code is in applicationEvaluator.R  Potentially refactor into base class or common method?

Reply:
Alternatively, mapFields could simply be a function defined in Utils.R and loaded via source("Utils.R") in applicationEvaluator.R, applicationTrainer.R, and applicationScorer.R.

Reply:
I did put it in another class, but we have data load in all three classes, so I had to put it after every load.

Reply:
Refactored and reduces amount of code for changing column names, and then I removed Utils class since there is no need now with this refactored code.